test/fixtures/stitch_mongo_fixture.js

```diff
@@ -52,22 +52,27 @@ export default class StitchMongoFixture {
     // create an app `test_app`, and authorize a user
     this.admin = new stitch.Admin('http://localhost:7080');
     await this.admin.client.authenticate('apiKey', userData.apiKey.key);
+
     const appConfig = EJSON.parse(fs.readFileSync(`${CONF_PATH}/test_app_config.json`, 'utf8'));
     let result = await this.admin.apps(userData.group.groupId).create(appConfig);
+    const mdb1 = appConfig.services.mdb1;
 
     this.clientAppId = result.clientAppId;
     await this.admin.apps(userData.group.groupId).app(result._id).authProviders().provider('api', 'key').update({});
     this.appKey = await this.admin.apps(userData.group.groupId).app(result._id).apiKeys().create({ name: 'app-key' });
     await this.admin.apps(userData.group.groupId).app(result._id).services().create({name: 'mdb1', type: 'mongodb'});
-    await this.admin.apps(userData.group.groupId).app(result._id).services().service('mdb1').setConfig(appConfig.services.mdb1.config);
+    await this.admin.apps(userData.group.groupId).app(result._id).services().service('mdb1').setConfig(mdb1.config);
+
+    const rule1Key = '5873a33f772e2e08ce645b9a';
+    const rule2Key = '5873a33f772e2e08ce645b9b';
 
-    const rule1 = await this.admin.apps(userData.group.groupId).app(result._id).services().service('mdb1').rules().create(appConfig.services.mdb1.rules['5873a33f772e2e08ce645b9a']);
-    appConfig.services.mdb1.rules['5873a33f772e2e08ce645b9a']._id = rule1._id;
-    await this.admin.apps(userData.group.groupId).app(result._id).services().service('mdb1').rules().rule(rule1._id).update(appConfig.services.mdb1.rules['5873a33f772e2e08ce645b9a']);
+    const rule1 = await this.admin.apps(userData.group.groupId).app(result._id).services().service('mdb1').rules().create(mdb1.rules[rule1Key]);
```

Comment:
actually - i unfortunately didn't think of this in my previous comment - but you can also reduce it even slightly further by doing something like
`const mdb1Svc = this.admin.apps(userData.group.groupId).app(result._id).services().service('mdb1')`

then all the lines below it just become `await mdb1Svc.rules().rule(...) <etc>`